meta-sokol-flex-common/classes/sdk_codebench_metadata.bbclass

```diff
@@ -24,7 +24,17 @@ CODEBENCH_SDK_VARS += "\
     SDK_TITLE \
     SDK_VERSION \
     gdb_serverpath=${bindir}/gdbserver \
+    TOOLCHAIN_PATH \
+    TOOLCHAIN_GDB_PATH \
 "
+
+TOOLCHAIN_PATH = "${SDKPATHNATIVE}${bindir_nativesdk}/${TARGET_SYS}"
+TOOLCHAIN_GDB_PATH = "${TOOLCHAIN_PATH}/${TARGET_PREFIX}gdb"
+
+SDKPATHTOOLCHAIN ?= "${SDKPATH}/toolchain"
```

Comment:
The intention here is this is the path into which we'll be redistributing the external Sourcery toolchain. That isn't currently being done, but once it is, the paths will be correct for external toolchain builds as well.